src/nvim/message.c

```diff
@@ -2234,15 +2234,11 @@ void mch_errmsg(char *str)
 {
   int len;
 
-#if (defined(UNIX) || defined(FEAT_GUI)) && !defined(ALWAYS_USE_GUI)
+#if defined(UNIX)
   /* On Unix use stderr if it's a tty.
    * When not going to start the GUI also use stderr.
    * On Mac, when started from Finder, stderr is the console. */
-  if (
-# ifdef UNIX
-    isatty(2)
-# endif
-    ) {
+  if (isatty(2)) {
```

Comment:
By "without any other check" I mean 'I don't now what really do ```os_isatty``` instead of ```isatty``` so I asked if I really can do this without any other check.

isatty replaced in commit a083983